site/app/views/grading/ElectronicGraderView.php

```diff
@@ -315,7 +315,7 @@ public function statusPage(
             "bulk_stats_url" => $this->core->buildCourseUrl(['gradeable', $gradeable->getId(), 'bulk_stats']),
             "details_url" => $details_url,
             "grade_url" => $this->core->buildCourseUrl(['gradeable', $gradeable->getId(), 'grading', 'grade']),
-            "regrade_allowed" => $this->core->getConfig()->isRegradeEnabled(),
+            "regrade_allowed" => false,
```

Comment:
Is there a reason why this parameter can't be removed entirely?

Reply:
Regrade_allowed is the course-specific grade inquiry setting, not the global setting. The global setting was removed. Unless you're specifically referring to the file ElectronicGraderView? To be honest I don't know much about this file; I think it can be removed as the issue never really came up in my testing until the unit tests online were done.

Reply:
After looking into the issue further, I think leaving this as-is is probably fine for now.  I was primarily concerned about the fact that we were just passing a constant to a Twig file, but it might be a challenge to simplify the logic so I think it's fine.